src/app/public/testing/colorpicker-fixture.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Allows interaction with a SKY UX colorpicker component.
+ */
+export class SkyColorpickerFixture {
+
+  private debugEl: DebugElement;
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(this.fixture, this.skyTestId, 'sky-colorpicker');
+  }
+
+  /**
+   * The colorpicker's currently selected color formatted to the `outputFormat`.
+   */
+  public get value(): string {
+    return this.getColorpickerInputEl().nativeElement.value;
+  }
+
+  /**
+   * Set the colorpicker's color hex code.
+   * @param hexValue The new color hex code. Must inculde '#'.
+   */
+  public async setColorFromHexValue(hexValue: string): Promise<any> {
+    await this.clickColorpickerButtonEl();
+
+    const hexInput = document.querySelector('input[id^=sky-colorpicker-hex-]') as HTMLInputElement;
+
+    hexInput.value = hexValue;
+    SkyAppTestUtility.fireDomEvent(hexInput, 'input');
+
+    await this.clickColorpickerApplyButtonEl();
+
+    return this.fixture.whenStable();
+  }
+
+  /**
+   * Set the colorpicker's color RGB values.
+   * @param r The red color value.
+   * @param g The green color value.
+   * @param b The blue color value.
+   */
+  public async setColorFromRGBValue(r: number, g: number, b: number): Promise<any> {
```

Comment:
Single-character parameters should be avoided on public methods because they can cause confusion to the consumer (most people know what "RGB" stands for, but there might be a few that don't). I'd go with `red`, `green`, and `blue`, instead.

Also, what do you think about supporting the alpha channel for MVP? We could make a fourth optional argument named `alpha?: number`.